wadf.conf

```diff
@@ -124,6 +124,9 @@ db_name = @underscore:instance@@db_number@
 db_user = @underscore:instance@
 db_pass = @underscore:instance@pass
 db_type = mysql
+; the following setting allows connections to remote mysql8+ servers
```

Comment:
Is there a reason why this only applies to MySQL 8+?

Reply:
If you check the deployDatabase() function in WADF.php, for mysql < 8 we still only use `GRANT ALL on {$name}.* to {$user} IDENTIFIED BY '{$pass}'`, where you don't need to specify the host; but in MySQL8 you need to create a user with the host specified before adding privileges, because `GRANT` does not support `IDENTIFIED BY` anymore.

Reply:
OK, my point was that if the new setting is introduced, it should apply consistently, across the board. Right now, it is only used for MySQL 8+ servers